instrumentation/action_mailer/lib/opentelemetry/instrumentation/action_mailer/railtie.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module ActionMailer
+      SUBSCRIPTIONS = %w[
+        deliver.action_mailer
+      ].freeze
+
+      # This Railtie sets up subscriptions to relevant ActionMailer notifications
+      class Railtie < ::Rails::Railtie
+        config.after_initialize do
+          ::OpenTelemetry::Instrumentation::ActiveSupport::Instrumentation.instance.install({})
+
+          SUBSCRIPTIONS.each do |subscription_name|
+            config = ActionMailer::Instrumentation.instance.config
+            ::OpenTelemetry::Instrumentation::ActiveSupport.subscribe(
```

Comment:
By omitting the email address, like omitting `:to`, `:from`, `:bcc`, `:cc`?
For semantic convention, I can only find the message_id from opentelemetry-ruby `MESSAGE_ID`. I didn't find anything that's related in spec.

Reply:
> By omitting the email address, like omitting :to, :from, :bcc, :cc?

I have updated the instrumentation that have modifying the options if email_address is omit; but not sure if it's best practice.

> rails.action_mailer.message.id

Do you mean like introduce a new semconv (i.e. RAILS_ACTION_MAILER_MESSAGE_ID=...)?

Reply:
Yes, I do mean introduce new semconv like I did for active job.

It may be something to ask in the semantic conventions SIG to see what others think.

Reply:
The problem is I have no idea what would be good names for this.